### DISCUSSION TITLE
add python CLI for duckdb

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#9569] [Ideas]
to simplify the query. 
something like https://github.com/chdb-io/chdb-cli

**Comment by Tishj** [comment]
We have a CLI already

**Comment by l1t1** [comment]
pip install duckdb --pre is easier to get the latest build, while CLI need to download and unzip manually.

**Reply by carlopi** [reply]
One very experimental approach you can try is this: https://github.com/carlopi/duckdb-latest#one-liner-install-script

Idea is that you run the script once, then duckdb-latest to run latest compiled duckdb (updated after nightly builds) or duckdb-update to re-download latest binary.

It sort of works, but it's a very side project by me, give it a try.
Or write your own equivalent script to wrap `wget  + unzip + place it somewhere`.

**Reply by l1t1** [reply]
thanks

**Comment by szarnyasg** [comment]
We reserve pip for the Python client and recommend downloading and unzipping DuckDB manually. There are packages for brew (macOS) and winget (Windows). Eventually Linux packages (apt/yum/etc.) will come along but it's unlikely that we'll use pip for installing the CLI client.
